numpy/distutils/intelccompiler.py

```diff
@@ -2,7 +2,7 @@
 
 from distutils.unixccompiler import UnixCCompiler
 from numpy.distutils.exec_command import find_executable
-from numpy.distutils.msvc9compiler import MSVCCompiler
+from distutils.msvc9compiler import MSVCCompiler
```

Comment:
``.distutils`` looks incorrect to me - there is no file/module ``msvc9compiler`` in numpy at all, it really does come from Python distutils. Why this isn't completely broken right now I don't know. IIRC we had this import in another place and fixed it recently.